docs/introduction.rst

```diff
@@ -1,73 +1,57 @@
 .. include:: defs.rst
 .. _introduction:
 
+********
 Overview
-========
-|AppName| is open source scientific image processing software integrating hardware control, data acquisition, visualization, processing, and analysis using Python. |AppName| is easily extended using Python. It runs on Windows, Linux, and macOS.
+********
+Nion Swift is an image processing software for controlling instruments, and managing the acquisition, organization, visualization, and analysis of data. The software was designed to be used in conjunction with Nion's electron microscopes, but the functionality and the ability to extend its capabilities using Python mean that Nion Swift is useful for any number of data visualization projects. Whether you already have your data, or you want to see the data as it is acquired, Nion Swift helps you visualize your data in exactly the way you want.
 
-|AppName| is intended to provide an extensible platform to acquire scientific data from a variety of microscopes and other instruments, be intuitive and easy to use, and allow sophisticated data visualization, processing, and analysis.
+Click to download Nion Swift or for :ref:`installation` instructions.
 
-Key Features
+Click here to get started fast with our :ref:`basic-use`.
 
-- Data handling of 1D plots, 2D images, 1D and 2D collections of plots and images, and sequences.
-- Live computations that can be chained during acquisition or live parameter adjustment.
-- Open source, cross platform (macOS, Windows, Linux), and Python based scientific data processing.
-- Low latency, high throughput data acquisition platform.
+User Interface
+==============
+The user interface is split up into a central section of the window called the workspace and the surrounding area for utility panels. 
```

Comment:
This could be reworded. "The user interface window is comprised of a central section called the workspace and individual utility panels typically attached to the left/right edges of the window." I say 'typically' because they can be floating. But we can gloss over that point for now.